code/addons/docs/README.md

```diff
@@ -4,13 +4,13 @@
 
```

Comment:
I made minimal edits here, but it kinda seems like this README should just include installation instructions and then just point to docs site. Then (later), then entire `code/addons/docs/docs/` directory could likely receive similar treatment. Wdyt?

Reply:
WFM